### PR TITLE
fix: [property-dialog] the device name show error.

### PR DESCRIPTION
**Pull request by GongHeng2017**
1. the devece name show error in device property dialog, and fix it.
2. adjust the UI

Log: fix issue
Bug: https://pms.uniontech.com/bug-view-214867.html

### CODE REVIEW

**Review by max-lvs — DISMISSED** [review]

File: src/plugins/filemanager/core/dfmplugin-computer/dfmplugin_computer_global.h [hunk 1]
```diff
@@ -26,6 +26,7 @@ struct DeviceInfo
     QString fileSystem;
     qint64 totalCapacity;
     qint64 availableSpace;
+    QString fileBaseName;
```
Comment (itsXuSt) [inline src/plugins/filemanager/core/dfmplugin-computer/dfmplugin_computer_global.h]: deviceDesc // 设备描述符

File: src/plugins/filemanager/core/dfmplugin-computer/utils/computerutils.cpp [hunk 2]
```diff
@@ -289,6 +289,7 @@ QWidget *ComputerUtils::devicePropertyDialog(const QUrl &url)
     devInfo.fileSystem = info->extraProperty(GlobalServerDefines::DeviceProperty::kFileSystem).toString();
     devInfo.totalCapacity = info->sizeTotal();
     devInfo.availableSpace = info->sizeFree();
+    devInfo.fileBaseName = info->baseName();
```
Comment (itsXuSt) [inline src/plugins/filemanager/core/dfmplugin-computer/utils/computerutils.cpp]: baseName 的转换过于隐匿。直接使用 extraProperty 获取 kDevice 属性。

**Review by itsXuSt — CHANGES_REQUESTED** [review]

File: src/plugins/filemanager/core/dfmplugin-computer/utils/computerutils.cpp (hunk 2 above)
Comment (GongHeng2017) [reply src/plugins/filemanager/core/dfmplugin-computer/utils/computerutils.cpp]: done

File: src/plugins/filemanager/core/dfmplugin-computer/dfmplugin_computer_global.h (hunk 1 above)
Comment (GongHeng2017) [reply src/plugins/filemanager/core/dfmplugin-computer/dfmplugin_computer_global.h]: done

**Review by itsXuSt — APPROVED** [review]

**Review by max-lvs — APPROVED** [review]

**Pull request merged**